synapse/lib/remcycle.py

```diff
@@ -344,6 +346,17 @@ def __repr__(self):
     def _runIoLoop(self):
         self.web_loop.start()
 
+    def _consume_queue(self):
+        '''
+        Callback for consuming objects from the web_queue in the IOLoop.
+        '''
+        while True:
+            obj = self._web_queue.get(timeout=0)
+            if obj is None:
```

Comment:
i try to avoid "is None" because of the possibility that a module in C instantiates and returns an instance of None rather than using the singleton None.  ( "is" means that their object id (memory address) is the same, which would not be true if a poorly implemented C module returned a newly created instance of None )